src/model/project.cpp

```diff
@@ -199,11 +199,7 @@ QString Project::detectEncoding() {
 	if (enc.empty()) {
 		std::ifstream i(findFile(RM_DB).toStdString(), std::ios::binary);
 		if (i) {
-			std::string dbenc = lcf::ReaderUtil::DetectEncoding(i);
-			lcf::Encoder encoder(dbenc);
-			if (encoder.IsOk()) {
-				enc = dbenc;
-			}
+			enc = lcf::ReaderUtil::GetEncoding(i);
```

Comment:
This is wrong since it operates on the `ini` file.
`DetectEncoding` should be the right function.

Reply:
The problem is that `DetectEncoding` no longer accepts `std::ifstream`. This is the reason why this PR was made in the first place.

Reply:
right, you need to load the db beforehand.
See e.g. Player: https://github.com/EasyRPG/Player/blob/a003a867c3edea437a21db698662c55c503b89eb/src/player.cpp#L1224

Reply:
Should be fixed now.